test/index.js

```diff
@@ -1124,6 +1124,29 @@ describe('expect()', () => {
                 Hoek.assert(!exception, exception);
                 done();
             });
+
+            it('asserts called with only one argument', (done) => {
+
+                let exception0 = false;
```

Comment:
Instead of `exception0`, `exception2`, `exception4`, etc., can you do:

```javascript
{
  let exception = false;
  try {
    ...
  }
}
```

i.e., separate the subtests with block scopes.